pyquil/__init__.py

```diff
@@ -1,3 +1,5 @@
-from pyquil.version import __version__
+from pyquil._version import pyquil_version
```

Comment:
At first, I had deleted this module, but I had to leave it as a "leaf" module to avoid some import cycles. So I just made it internal.